pages/profiles/sahityaLMaruvada.md

```diff
@@ -0,0 +1,26 @@
+# Lalitha Sahitya Maruvada
+## Other Details
+  **TimeZone:**   Eastern Standard Time  
+  **Location:**   Maryland  
+  **OS:**         Ubuntu 16.04 LTS  
+
+## About Me!
+I am graduate student in University of Maryland specializing in Networking. My primary areas of interest include Software Development,Cloud Computing,IoT. I am excited to work on this project as I really liked the idealogy behind the organization and its contribution to the kids in low resource areas.
+
+**Coding Interests:**  
+
+Languages | Frameworks
+------------ | ------------
+Python  | Drupal
+Java  |  Ruby On Rails
+HTML,CSS| Wordpress
+Ruby|
+
+I am very much interested in exploring the domain of Cloud Computing. My personal favourite hobbies include Traveling, Cooking.
+
+---
+Please feel free to connect with me on  
+*[LinkedIn](https://www.linkedin.com/in/lalitha-sahitya-maruvada/)  
```

Comment:
You did a great job, however, in order to create a bullet point, you need to add space after the `*` symbol 😉